metadata.json

```diff
@@ -52,7 +52,9 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "14.04"
+        "14.04",
```

Comment:
Drop 14?
Also: metadata.json isn't calling for apt as a requirement: deliberate?  And, any interest in adding Debian to the mix instead of just Ubuntu?

Reply:
Removed 14.04, added Debian 9.

Reply:
Added apt as dependency, had missed staging the change.